.codesandbox/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:lts-alpine
+
+# 1. zsh is default shell on CodeSandbox - without it, the devbox fails to start
+# with a "OCI runtime exec failed" error:
+# ```
+# OCI runtime exec failed: exec failed: unable to start container process: exec: '/bin/zsh': stat /bin/zsh: no such file or directory: unknown
+# ```
+#
+# 2. Git is a dependency of zsh - without it, opening the terminal fails with a
+# "command not found" error:
+# ```
+# "/root/.oh-my-zsh/tools/check_for_upgrade.sh:31: command not found: git"
+# ```
+#
+# 3. yq used to install matching version of pnpm
+#
+# TODO: Switch to bash and remove the zsh packages in future if CodeSandbox adds
+# a feature to configure the shell
+# https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294#discussion_r1601759357
+
+# TODO: Switch to bash and remove the zsh packages in future if CodeSandbox adds
+# a feature to configure the shell
+# https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294#discussion_r1601759357
```

Comment:
duped

Reply:
Didn't check the diff. Deleted the duped comment
[aa37615](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/aa376153521a0ec4a40d9eb42422ac0d08d2c4eb)